executor/include/execution/trait/can_prefer.hpp

```diff
@@ -24,7 +25,7 @@ struct can_prefer : std::false_type {};
 
 template <typename Executor, typename Property>
 struct can_prefer<Executor, Property,
-                  std::void_t<decltype(execution::require(
+                  detail::void_t<decltype(execution::require(
```

Comment:
PCL has void_t IIRC

Reply:
Using a macro for the namespace now. Will replace the macro with the pcl namespace when integrating.